src/content/ui-api/kubectl-gs/template-app.md

```diff
@@ -68,6 +68,7 @@ It also supports the following optional flags:
 - `--user-secret`: Path to the user secrets YAML file.
 - `--namespace-annotations`: Additional annotations to be appended to the metadata of the target namespace (through [`spec.namespaceConfig.annotations`]({{< relref "/app-platform/namespace-configuration/index.md" >}}) of [App]({{< relref "/ui-api/management-api/crd/apps.application.giantswarm.io.md" >}}) CR) in form `key=value`. To specify multiple annotations, either separate annotation pairs with commata (,) or specify the flag multiple times.
 - `--namespace-labels`: Additional labels to be appended to the metadata of the target namespace (through [`spec.namespaceConfig.labels`]({{< relref "/app-platform/namespace-configuration/index.md" >}}) of [App]({{< relref "/ui-api/management-api/crd/apps.application.giantswarm.io.md" >}}) CR) in form `key=value`. To specify multiple labels, either separate label pairs with commata (,) or specify the flag multiple times.
+- `--in-cluster`: Creates in-cluster App CR, necessary for insalling collection of apps, for example [Security Pack]({{< relref "/app-platform/apps/security/index.md" >}}).
```

Comment:
App CR is a link in the other lines above, maybe add it here?

Reply:
Sure, makes sense 👍